locales/cy/translation.json

```diff
@@ -491,6 +491,23 @@
         }
       }
     },
+    "pyiTriageAnotherWay": {
+      "title": "You need to use the GOV.UK One Login app to prove your identity",
+      "header": "You need to use the GOV.UK One Login app to prove your identity",
+      "content": {
+        "paragraph1": "If you do not live in the UK, you can only use the app and a smartphone to prove your identity with GOV.UK One Login.",
+        "subHeading1": "What would you like to do?",
+        "formRadioButtons": {
+          "continueToServiceButtonText": "Go to the service you were trying to use to look for other ways to access it",
```

Comment:
Figma has `Continue` instead of `Go`

Reply:
The figma actually uses `Go`, the screenshot which tends to get outdated quick uses continue. So I think `Go` is correct here